app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
```

Comment:
You could perform unpacking during the split operation, in this way, you will avoid using indexing and make your code more descriptive:
```suggestion
    command_name, source_file, destination, *_ = command.split()
```